inference-engine/tests/ngraph_functions/CMakeLists.txt

```diff
@@ -18,6 +18,10 @@ addIeTarget(
         ROOT ${PUBLIC_HEADERS_DIR}
         ADDITIONAL_SOURCE_DIRS
             ${CMAKE_CURRENT_SOURCE_DIR}/src
+        LINK_LIBRARIES
+            funcTestUtils
+            # this import as result of including nGraph function building for tests
+            inference_engine_transformations
```

Comment:
Why do we need to link against these libraries. I believe ngraph_functions part shouldn't depend on them.

Reply:
@dmitry-gorokhov For unknown reasons, inference-engine\tests\ngraph_functions\include\ngraph_functions\subgraph_builders.hpp is not compiled because of missing include paths. This lines are fixed it. As I can see, this header never used before in ngraphFunctions target.

Reply:
When I believe you just need to change `#include "ngraph_functions/subgraph_builders.hpp"` on `#include "ngraph_functions/builders.hpp"` in multiply_function.cpp. That should resolve the issue.